### PR TITLE
HCCDOC-2557: Adding a Doc tile to OCP Lib 

### PR DESCRIPTION
**Pull request by jseseCCS**
[HCCDOC-2557](https://issues.redhat.com/browse/HCCDOC-2557): Adding a Doc tile to OCP Lib from OS Learning Resources page

### CODE REVIEW

**Comment by jseseCCS** [comment]
@Hyperkid123 hi, Martin! i hope you can review this for accuracy, etc. thanks!

**Comment by Hyperkid123** [comment]
Hi @jseseCCS there is no assiciated metadata file for the tile. It will not be "discovered" and seeded. The initial setup of a new tile is described here: https://github.com/RedHatInsights/quickstarts/blob/main/docs/quickstarts/README.md#create-a-working-branch-and-make-your-docs-updates

**Comment by jseseCCS** [comment]
i knew i'd forgotten something, @Hyperkid123! thanks. adding now.

**Comment by jseseCCS** [comment]
@Hyperkid123 here's an image grab of my folder structure with this yaml file. do I have to put it in its own folder and give it its own metadata file or can I keep it in the same folder (like you see in this image) and add two names to the metadata file?

Screenshot from 2024-06-12 10-28-06

**Comment by jseseCCS** [comment]
@Hyperkid123 take another look? it seems like I made it worse but I can't imagine how!

File: docs/quickstarts/openshift-ocp-2/openshift-ocp-library.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+apiVersion: console.openshift.io/v1
+kind: QuickStarts
+metadata:
+  name: openshift-ocp-library
```
Comment (Hyperkid123) [inline docs/quickstarts/openshift-ocp-2/openshift-ocp-library.yml:4]: This name as to be `openshift-ocp-2`

It needs to match the name from the metadata file.

**Review by Hyperkid123 — COMMENTED** [review]
Just one small change and I think this can be merged :slightly_smiling_face:

File: docs/quickstarts/openshift-ocp-2/openshift-ocp-library.yml (hunk 1 above)
Comment (jseseCCS) [reply docs/quickstarts/openshift-ocp-2/openshift-ocp-library.yml:4]: ohhhhhhhhh ok! fixing

**Comment by jseseCCS** [comment]
@Hyperkid123 still getting all these errors. what am I doing wrong? and thank you so much for all your help!

**Comment by Hyperkid123** [comment]
@jseseCCS  The filename did not match. The security errors can be ignored. I've pushed the filename change. I'll merge the PR once I get thumbs up from you :slightly_smiling_face: .

**Comment by jseseCCS** [comment]
@Hyperkid123 please go for it and thank you!!

**Comment by jseseCCS** [comment]
tysm, @Hyperkid123!

**Pull request merged**